javascript-modal/main.js

```diff
@@ -0,0 +1,11 @@
+var modal = document.getElementById("theModal");
+var button = document.getElementById("open");
+var noButton = document.getElementById("no");
+
+button.onclick = function (event) {
+  modal.style.display = "block";
+
+  noButton.addEventListener("click", function (event) {
```

Comment:
you should not add the event listener to the button inside of the function which is called to show them modal because it is adding another event listener everytime the function is called. Please update!